modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsDetails/index.tsx

```diff
@@ -0,0 +1,209 @@
+		{
+			active: currentStep === STEP.REPAIR,
+			complete: currentStep > STEP.REPAIR,
+			show: true,
+			title: setFirstLetterUpperCase(
+				CONSTANTS.CLAIM_STATUS['repair'].NAME
+			),
+		},
+		{
+			active: currentStep === STEP.PEDDINGSETTLEMENT,
+			complete: currentStep > STEP.PEDDINGSETTLEMENT,
+			show: true,
+			title: setFirstLetterUpperCase(
+				CONSTANTS.CLAIM_STATUS['pendingSettlement'].NAME
+			),
+		},
+	];
+
+	const selectCurrentStep = (claimStatus: string) => {
+		const status = CONSTANTS.CLAIM_STATUS[claimStatus].INDEX;
+
+		setCurrentStep(status);
+	};
+
+	useEffect(() => {
+		const queryParams = new URLSearchParams(window.location.search);
+		const claimId = Number(Array.from(queryParams.values())[0]);
+
+		if (claimId) {
+			getClaimsData(claimId).then((response) => {
+				const claimData = response?.data;
+
+				const claimStatus = claimData?.claimStatus?.key;
+
+				if (claimStatus === 'settled') {
+					setIsClaimSettled(true);
+				}
+
+				selectCurrentStep(claimStatus);
+
+				setClaimData(claimData);
+			});
+		}
+
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, []);
+
+	const applicationData =
+		claimData?.r_policyToClaims_c_raylifePolicy
+			?.r_quoteToPolicies_c_raylifeQuote
+			?.r_applicationToQuotes_c_raylifeApplication;
+
+	const fullName = applicationData?.firstName
+		? `${applicationData?.firstName} ${applicationData?.lastName}`
+		: applicationData?.firstName;
+
+	const summaryClaimData: ClaimDetailDataType[] = [
+		{
+			data: dateFormatter(claimData?.claimCreateDate),
+			key: 'submittedOn',
+			text: 'Submitted on',
+		},
+		{
+			data: claimData?.r_policyToClaims_c_raylifePolicyId,
+			icon: true,
+			key: 'entryID',
+			redirectTo: `${'policy-details'}?externalReferenceCode=${
+				claimData?.r_policyToClaims_c_raylifePolicyERC
+			}`,
+			text: 'Policy Number',
+			type: 'link',
+		},
+		{
+			data: fullName,
+			key: 'name',
+			text: 'Name',
+		},
+		{
+			data: applicationData?.email,
+			key: 'email',
+			redirectTo: applicationData?.email,
+			text: 'Email',
+			type: 'link',
+		},
+		{data: applicationData?.phone, key: 'phone', text: 'Phone'},
+	];
+
+	const summaryClaimDataSettled: ClaimDetailDataType[] = [
+		{
+			data: claimData?.claimStatus?.name,
+			greenColor: true,
+			key: 'status',
+			text: `Status`,
+		},
+		{
+			data: dateFormatter(claimData?.settledDate),
+			key: 'settledOn',
+			text: `Settled on`,
+		},
+		{
+			data: currencyFormatter(claimData?.claimAmount),
+			key: 'settlementAmount',
+			text: `Settlement Amount`,
+		},
+		...summaryClaimData,
+	];
+
+	return (
+		<div className="claim-details-container">
+			{!isClaimSettled && (
+				<div className="align-items-center bg-neutral-0 d-flex justify-content-center multi-steps-content">
+					<MultiSteps steps={steps.filter((step) => step.show)} />
+				</div>
+			)}
+
+			<div className="claim-detail-content d-flex py-4 row">
+				<div className="col-12 col-lg-12 col-md-12 col-sm-12 col-xl-3 d-flex mb-4">
```

Comment:
Are all these classes really necessary? Bc we're using 3 types of columns here, right? Large, middle and small

Reply:
You're right, they're not necessary.